CODEOWNERS

```diff
@@ -0,0 +1 @@
+*       @Guillermina22 @juan-apa @micaelalorenzo
```

Comment:
missing new line here. If possible configure your IDE so that a new empty line is added at the EOF on save.
https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files#:~:text=insertFinalNewline%22%3A%20true%20%2C%20within%20the,there%20isn't%20already%20one.